scripts/masterCron.ts

```diff
@@ -211,6 +249,47 @@ async function getAnnualizedVol(underlying: string, resolution: number) {
   return candles[candles.length - 1][pricePoint] * 10 ** 6;
 }
 
+async function updateTokenList(
+  fileName: string,
+  vaultArtifactAbi: any,
+  ierc20Abi: any,
+  provider: any,
+  network: string
+) {
+  let tokens = [];
```

Comment:
Shouldn't we read the original list from the file and append to it? Instead of creating an empty array each time

Reply:
I think we should create an empty array each time. previous list does not matter since those otokens are irrelevant at this point. think it should just be the otokens relevant for this current work instead of a neverending list.

Reply:
Oh yeah, I didn't consider this. You're right.

Reply:
My take on this is that we should have a never-ending list because users can browse through the past auctions and see the Ribbon symbol in them.

Reply:
fixed